File: api/RS.API/RS.Model/Approval/ApprovalEventViewModel.cs
using RS.ViewModel.User;
using System;
using System.Collections.Generic;
using System.Text;

namespace RS.ViewModel.Approval
{
    public class ApprovalEventViewModel
    {
        public int ApprovalEventId { get; set; }

        public string ApprovalEventName { get; set; }

        public int ApprovalEventOrder { get; set; }

        public int ApprovalId { get; set; }

        public List<ApprovalActionViewModel> ApprovalActions { get; set; }

        public List<UserViewModel> Users { get; set; }

        // public List<ApprovalEventRoles> ApprovalEventRoles { get; set; }
    }
}

File: web/src/app/webapi/models/approval-event-view-model.ts
/* tslint:disable */
import { ApprovalActionViewModel } from './approval-action-view-model';
import { UserViewModel } from './user-view-model';

/**
 */
export class ApprovalEventViewModel {
    approvalEventId?: number;
    approvalEventName?: string;
    approvalEventOrder?: number;
    approvalId?: number;
    approvalActions?: ApprovalActionViewModel[];
    users?: UserViewModel[];
}
